src/rtp.c

```diff
@@ -863,7 +863,7 @@ void janus_rtp_header_update(janus_rtp_header *header, janus_rtp_switching_conte
 	}
 	if(context->ts_reset) {
 		/* Video timestamp was paused for a while */
-		JANUS_LOG(LOG_HUGE, "Video RTP timestamp reset requested");
+		JANUS_LOG(LOG_HUGE, "Video RTP timestamp reset requested\n");
```

Comment:
I guess we should also remove the word "Video" from that log line, since now contexts are now media-type-agnostic.